### PR TITLE
Db rails outside container

### PR DESCRIPTION
**Pull request by devonbush**
This:
 1. enables local non-docker development in rails/js
 2. Enables live reload for files in the app/javascript directory

Ideally, all future SCSS and JS development is in the app/javascript directory, and app/assets is only used for third party static assets.

Note: line 314 is the only substantive README change, the rest is just trailing whitespace cleanup

### CODE REVIEW

File: NON_CONTAINERIZED_DEV_README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Developing on SCP without a docker container
+
+Developing on SCP without a Docker container, while less robust, opens up some faster development paradigms, including live css/js reloading, faster build times, and byebug debugging in rails.
+
+## SETUP
+
+1.  Run ruby -v to ensure Ruby 2.5.7 is installed on your local machine.  If not,  install rvm, then rvm install 2.5.7, then rvm use 2.5.7.
+2.  Run bundler -v to ensure Bundler is installed.  If not,  gem install bundler.
```
Comment (eweitz) [inline NON_CONTAINERIZED_DEV_README.md]:
```suggestion
2.  Run `bundler -v` to ensure Bundler is installed.  If not, `gem install bundler`.
```

File: NON_CONTAINERIZED_DEV_README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Developing on SCP without a docker container
+
+Developing on SCP without a Docker container, while less robust, opens up some faster development paradigms, including live css/js reloading, faster build times, and byebug debugging in rails.
+
+## SETUP
+
+1.  Run ruby -v to ensure Ruby 2.5.7 is installed on your local machine.  If not,  install rvm, then rvm install 2.5.7, then rvm use 2.5.7.
```
Comment (eweitz) [inline NON_CONTAINERIZED_DEV_README.md]:
```suggestion
1.  Run `ruby -v` to ensure Ruby 2.5.7 is installed on your local machine.  If not, [install rvm](https://rvm.io/rvm/install), then `rvm install 2.5.7`, then `rvm use 2.5.7`.
```

File: NON_CONTAINERIZED_DEV_README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+2.  Run bundler -v to ensure Bundler is installed.  If not,  gem install bundler.
+3.  cd to where you have the single_cell_portal_core Git repo checked out.
```
Comment (eweitz) [inline NON_CONTAINERIZED_DEV_README.md]:
```suggestion
3.  `cd` to where you have the `single_cell_portal_core` Git repo checked out.
```

File: NON_CONTAINERIZED_DEV_README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+4.  Run `bundle install`
+5.  Run `ruby rails_local_setup.rb $BROAD_USERNAME`, where $BROAD_USERNAME is a something like eweitz
+6.  Run the source command the script outputs
+7.  Run `rails s`
+8.  (optional, for live reload) In another tab, run bin/webpack-dev-server
```
Comment (eweitz) [inline NON_CONTAINERIZED_DEV_README.md]:
```suggestion
8.  (optional, for live reload) In another tab, run `bin/webpack-dev-server`
```

File: NON_CONTAINERIZED_DEV_README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+9. (needed if you are working on functionality that involves delayed jobs).
+    * In another terminal, run the source command output in step 7
+    * run `rails jobs:work`
+10.  You're all set!  You can now go to localhost:3000 and see the website.
+
+
+## KNOWN ISSUES
+0. Developing outside the docker container inherently runs more risk that your code will not work in the docker environment in staging/production.  BE CAREFUL.  If your changes are non-trivial, confirm your changes work in the containerized deploy before committing (ESPECIALLY changes involving package.json and/or the Gemfile)
```
Comment (eweitz) [inline NON_CONTAINERIZED_DEV_README.md]: Starting at 1 instead of 0 would be more consistent with our other documentation.

Comment (devonbush) [reply NON_CONTAINERIZED_DEV_README.md]: also more consistent with the list immediately above it :)

**Pull request closed without merging**